### PR TITLE
Search engine_version1_index

### PR DESCRIPTION
**Pull request by pkleungae**

### CODE REVIEW

**Comment by wai2016** [comment]
I think you have to squash the commit.

File: app/index.html [hunk 1]
```diff
@@ -7,37 +7,52 @@
...
+
+      <script src="lib/jquery.min.js"></script>
+      <script src="lib/bootstrap.min.js"></script>
+      <script src="lib/angular/angular.min.js"></script>
+      <script src="lib/angular/angular-route.min.js"></script>
+      <!-- Adding firebase -->
+      <script src="https://www.gstatic.com/firebasejs/3.6.0/firebase.js"></script>
+      <script>
+      // Initialize Firebase
+      var config = {
```
Comment (wai2016) [inline app/index.html]: You can initialize firebase in site.js

**Review by wai2016 — COMMENTED** [review]
Code review

File: app/index.html (hunk 1 above)
Comment (pkleungae) [reply app/index.html]: true

**Pull request merged**